libraries/ms-drivers/src/stm32f0xx/spv1020_mppt.c

```diff
@@ -0,0 +1,45 @@
+#include "spv1020_mppt.h"
+#include "spv1020_mppt_defs.h"
+
+// This file implements stm32-specific functionality. See ../spv1020_mppt.c for the rest.
+
+// Send a command to SPV1020 and get a response. |rx_len| must be <= 2.
+// It's safe to pass NULL as |rx_data| if |rx_len| is 0.
+static StatusCode prv_send_command(SpiPort port, uint8_t command, uint8_t *rx_data, size_t rx_len) {
+  if (port >= NUM_SPI_PORTS) {
+    return status_code(STATUS_CODE_INVALID_ARGS);
+  }
+
+  // We don't use spi_exchange since that assumes 0x00 is NOP, whereas SPV1020 appears to use 0x01.
+  spi_cs_set_state(port, GPIO_STATE_LOW);
```

Comment:
Why is the GPIO_STATE_LOW set twice for the same port?

Reply:
Whoops, fixed that